operator/pom.xml

```diff
@@ -30,7 +30,7 @@
       <plugin>
         <groupId>pl.project13.maven</groupId>
         <artifactId>git-commit-id-plugin</artifactId>
-        <version>2.2.6</version>
+        <version>${git-commit-id-plugin-version}</version>
```

Comment:
nit - shouldn't this be in the plugin management section instead of properties?

Reply:
Honestly, not sure. I just found another version number that wasn't with the others and wanted to fix that. I'm happy to see if I can move this to plugin management.

Reply:
Going to punt on this for now.